content/defects-tech-debt.rst

```diff
@@ -190,6 +190,16 @@ to go look for the details of the ticket or just ignore it, as it is not related
 to what you are doing now.
 
 
+When you find a bug 
```

Comment:
I think that this should start at line 68... before tech-debt.

The title can be "Reporting a bug"... here is my take on this :)

Each bug should be reported in a separate ticket.
The report should contain detailed steps for reproducing the bug.
Keep note of the version in which this bug was observed.
For bugs which are not yet released, try to reproduce them on master and make a not of the master version.

You can create a branch for the ticket which will contain the configuration use to reproduce the defect.
Publish the branch and make a note in the ticket report.